packages/output/src/test/page.js

```diff
@@ -1505,6 +1505,10 @@ describe('Page output', () => {
         backgroundColor: { color: { r: 255, g: 255, b: 255 } },
         page: {
           id: 'bar',
+          pageAttachment: {
+            theme: 'light',
+            ctaText: 'Buy now',
+          },
```

Comment:
Let's move this to a new test, e.g. `it('should render shopping attachment with custom CTA text`)

Also a good opportunity to add a new `it('should produce valid output with shopping products and custom CTA text')` test for AMP validation.

Aside: the existing `should produce valid output with shopping products` test is currently skipped, but I think now it can be enabled as it should work.